### DISCUSSION TITLE
[Feature Request] Short Video Generation

### DISCUSSION THREAD

**Discussion by Maxxxel** [hlky/stable-diffusion-webui#214] [Feature suggestions]
Just found this gist: https://gist.github.com/karpathy/00103b0037c5aaea32fe1da1af553355

Would be a nice addition 😁🤞

**Comment by jansteffen** [comment]
I gotta be honest, I think this is such a niche and gimmicky use case it's not worth bloating up the UI even more for.

**Comment by Maxxxel** [comment]
Well it's just another Tab

**Comment by hlky** [comment]
> I gotta be honest, I think this is such a niche and gimmicky use case it's not worth bloating up the UI even more for.

Feels that way to me too, but:

What if 'niche' features like this were separate standalone scripts?

@Maxxxel would that work for you?

**Comment by Maxxxel** [comment]
Ofc!

In my eyes it's as simple as adding the generated image to the script and add a few parameters in a Video tab, similar to image2image.
